Sample/Kitchen Sink/DemonstrationTools/UnitConverter.swift

```diff
@@ -0,0 +1,184 @@
+        .background(theme.color.background)
+        .navigationBarTitle(demonstrationVM.naivgationBarTitle)
+    }
+
+    var calculationResult: some View {
+        InputField(text: $demonstrationVM.outputValue, placeholder: demonstrationVM.calculatedField)
+            .textFieldStyle(.advisory)
+            .padding([.bottom], themeManager.current.padding.grid2x)
+    }
+
+    var lengthInput: some View {
+        InputField(text: $demonstrationVM.inputValue,
+                   placeholder: demonstrationVM.lengthPalceholder,
+                   bottomLabelConfig: BottomLabelConfig(""),
+                   filter: .doubleOnly)
+        .textFieldStyle(.default)
+    }
+
+    var lengthInputUnitPicker: some View {
+        MenuField(selection: $demonstrationVM.boundSelectionInput,
+                  options: UnitConverterViewModel.LengthType.allCases,
+                  placeholder: "")
+        .menuFieldStyle(.default)
+        .padding([.bottom], themeManager.current.padding.grid2x)
+    }
+
+    var lengthOutputUnitPicker: some View {
+        MenuField(selection: $demonstrationVM.boundSelectionOutput,
+                  options: UnitConverterViewModel.LengthType.allCases,
+                  placeholder: "")
+        .menuFieldStyle(.default)
+        .padding([.bottom], themeManager.current.padding.grid2x)
+    }
+
+    var kgInputField: some View {
+        InputField(text: $demonstrationVM.kgInputString,
+                   placeholder: demonstrationVM.kgHint,
+                   bottomLabelConfig: BottomLabelConfig(demonstrationVM.bottomKgLabel),
+                   formatter: { typedString in
+            guard let decimalValue = Decimal(string: typedString) else { return typedString }
+            return demonstrationVM.toString2DP(value: decimalValue)
+        }, filter: .doubleOnly)
+        .textFieldStyle(demonstrationVM.emptyFields ? DefaultTextFieldStyle.caution : DefaultTextFieldStyle.default)
+    }
+
+    var lbInputField: some View {
+        InputField(text: $demonstrationVM.lbsInputString,
+                   placeholder: demonstrationVM.lbHint,
+                   bottomLabelConfig: BottomLabelConfig(demonstrationVM.bottomlbLabel),
+                   formatter: { typedString in
+            guard let decimalValue = Decimal(string: typedString) else { return typedString }
+            return demonstrationVM.toString2DP(value: decimalValue)
+        }, filter: .doubleOnly)
+        .textFieldStyle(demonstrationVM.emptyFields ? DefaultTextFieldStyle.caution : DefaultTextFieldStyle.default)
+    }
+
+    var swapButton: some View {
+        Button(action: {
+            withAnimation(.easeInOut(duration: 0.5)) {
+                swapFields()
+            }
+        }, label: {
+            Image(systemName: "arrow.left.arrow.right")
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 40, height: 40)
```

Comment:
Minor: Instead of using `40` here, there's a size within FlightUI for icon sizes, probably using `theme.size.iconMedium` or `theme.size.iconLarge` would be good ones to use here 👍